docs/book/src/tasks/experimental-features/runtime-sdk/implement-extensions.md

```diff
@@ -67,15 +67,17 @@ var (
 )
 
 func init() {
-	// Register the Runtime Hook types into the catalog.
+    // Adds to the catalog all the RuntimeHooks defined in cluster API.
```

Comment:
tabs vs spaces!? also for the other lines?

Seems like all existing lines use tabs, the changed ones now use 4 spaces instead 🤔

Reply:
Good catch!

Let's use tabs consistently like in Go, otherwise the rendered formatting is off

Reply:
fixed, nice catch